css/index.css

```diff
@@ -51,6 +51,144 @@ table {
     box-sizing: border-box;
 }
 
+header {
+    width: 100%;
+    background: white;
+    display: flex;
+    align-items: baseline;
+    padding: 20px;
+    margin:auto;
+    justify-content:center;
+    flex-wrap: wrap;
+    
+}
+
+header nav {
+    width: 60%;
+    display: flex;
+    justify-content: space-between;
+    align-items: baseline;
+    padding: 20px 2%;
+    font-size: 14px;
+    margin-left: 10px;
+}
+
+header nav a {
+    color:black;
+    text-decoration: none;
+    font-weight: bold;
+    justify-content: space-around;
+}
+
+header img {
+    justify-content: center;
+    margin: 10px;
+    align-self: auto;
+   
+ 
+}
+
+
+.about{
+    border-bottom: 1px dashed black;
+}
+
+div {
+    
+    padding: 10px;
+   
+}
+
+.about-boxes {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content:space-around;
+    align-items: center;
+    padding: 10px;
+    
+  
+}
+
+.about-boxes div {
+    border-bottom: 1px dashed black;
+
+}
+
+.about-boxes button {
+    border: 2px solid black;
+    border-radius: 25px;
+    width: 125px;
+    height: 40px;
+    margin: 10px;
+}
+
+.about-boxes img {
+        margin: 5px;
+        width: 300px;
+        margin-bottom: 10px;
+      
+    }
+  
+.about-boxes p {
+      
+        text-align:left;
+        width: 300px;
+        margin-top: 10px;
+        margin-bottom: 10px;
+    }
+
+.about-boxes h2 {
+        width: 300px
+}
+
+
+.flex {
+    margin: 0;
+    display: flex;
+    border: 5px solid #333;
+    margin-bottom: 2rem;
+  }
+
+
+.box:nth-child(1) {
```

Comment:
Loved the way you used the "nth-child" CSS selector in order to color the boxes in a different color. You can also achieve this is by keeping the "box" class that assigns that specific element its shape and add an additional class to each box to specify the color of the box. For example:

```
<div class="boxes">
    <div class = "box box1">Box 1</div>
    <div class = "box box2">Box 2</div>
    <div class = "box box3">Box 3</div>
</div>
```
This allows you to reuse your CSS styles for the "box" class, while "box X" is the specified color.
Nonetheless, great job!